ui/common/src/main/res/drawable/ic_diversity_1.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="960"
+    android:viewportHeight="960">
+  <path
+      android:fillColor="#FF000000"
```

Comment:
Ah, could you please specify the attr here, so it works in both dark and light theme? See other settings icons for an example.

Reply:
Sure, thank you :)